samples/user-scope-web-application/nodejs/controller/notification-controller.js

```diff
@@ -0,0 +1,207 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { GraphHelper } = require("../helper/graph-helper");
+
+/** Check Existing Subscriotion */
+const checkExistingSubsription = async (req, res) => {
+    try {
+        await GraphHelper.checkExistingSubscription();
+        let existingResource = await GraphHelper.checkExistingSubscription();
+        console.log(req);
+
+        if (existingResource != null) {
+            console.log("EXISTING SUBSCRIPTION IS : ", existingResource);
+            res.status(202).send(existingResource);
+        }
+        else {
+            console.log("NO EXISTING SUBSCRIPTION FOUND");
+            res.status(202).send();
+        }
+    }
+    catch (ex) {
+        console.error(ex);
+        res.status(500).send();
+    }
+}
+
+/** Creates Subscription for Channel */
+const createChannelAsync = async (req, res) => {
+    var teamId = req.query.teamId;
+    var subsId = "1";
+
+    try {
+        await GraphHelper.createSubscription(teamId, subsId);
+        res.status(202).send();
+    }
+    catch (ex) {
+        console.error(ex);
+        res.status(500).send();
+    }
+}
+
+/** Creates Subscription for Team */
+const createTeamAsync = async (req, res) => {
+    var teamId = req.query.teamId;
+    var subsId = "2";
+
+    try {
+        var result = await GraphHelper.createSubscription(teamId, subsId);
+        res.status(202).send(result);
+    }
+    catch (ex) {
+        console.error(ex);
+        res.status(500).send();
+    }
+}
+
+/** Creates Subscription for UserSCope Chats */
+const subscribeToSpecificChat = async (req, res) => {
+    var chatId = req.query.chatId;
+    var subsId = "3";
```

Comment:
remove hardcodings from everywhere, and from where we are using subsId?

Reply:
fixed